packages/indexer/src/EventHandlers.ts

```diff
@@ -65,7 +60,7 @@ ShowHubContract_ConditionModuleWhitelisted_handler(({ event, context }) => {
 ShowHubContract_Created_handlerAsync(async ({ event, context }) => {
   const chainId = event.chainId
   const eventId = `${chainId}-${event.params.id}`
-  const moduleId = `${chainId}-${event.params.conditionModule}`
```

Comment:
I feel like there is risk of bugs when manually constructing the id each time.

I put these in: `packages/indexer/src/utils/entityId.ts` (and only did it for 2 entities, not all yet. Also not sure if this pattern will work for all entities but similar kinds of helper functions can be used to try prevent these kinds of errors at compile time)